k8s/namespaces/em/em-icp/em-icp.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: em-icp
+spec:
+  releaseName: em-icp
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/em-icp
+  values:
+    nodejs:
+      replicas: 1
```

Comment:
why is it only 1 replica and is it running on only 1 cluster ?

Reply:
because ICP needs sticky sessions and for the pilot we agreed that we'll just run it in one cluster to deal with the issue.

Reply:
May be a separate conversation for this, but i am shocked this made it through till prod.

Reply:
it was a pilot, so we agreed this workaround at the time with the RPE team, and the plan is to implement sticky sessions if/when this is taken further